android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Lingua</string>
+    <string name="facebook_app_id">524074511756798</string>
```

Comment:
probably we should find ways to hide such sensitive information. For now, it is no big deal.